.travis.yml

```diff
@@ -6,7 +6,6 @@ language: java
 install: ./gradlew assemble
 script:
   - "./gradlew check"
-  - "./gradlew -Ptests.timeout.multiplier=3 integrationTest -i"
```

Comment:
I'm not sure what '-i' does but I'm pretty sure that its not necessary anymore...